File: C#/teste-prato/teste-prato/Models/Pergunta.cs
﻿using System;
using System.Collections.Generic;
using System.Text;
using teste_prato.Utils;

namespace teste_prato.Models
{
    public class Pergunta
    {
        public Pergunta(Prato prato)
        {
            this.prato = prato;
        }

        public string pergunta { get { return String.Format("O prato que você pensou é {0}?", prato.nome); } }
        public Prato prato { get; set; }
        public FilaPerguntas filaPerguntas { get; set; }
    }
}

File: Java/teste-prato/src/br/com/pratoTest/Model/Pergunta.java
package br.com.pratoTest.Model;

import br.com.pratoTest.utils.FilaPerguntas;

public class Pergunta {
	private String pergunta = "O prato que você pensou é %s?";
	private Prato prato;
	private FilaPerguntas filaPerguntas = new FilaPerguntas(); 
	
	public Pergunta() {
	}
	
	public Pergunta(Prato prato) {
		this.prato = prato;
	}
	
	public String getPergunta() {
		return String.format(pergunta, prato.getNome());
	}
	public Prato getPrato() {
		return prato;
	}
	public void setPrato(Prato prato) {
		this.prato = prato;
	}

	public FilaPerguntas getFilaPerguntas() {
		return filaPerguntas;
	}

	public void setFilaPerguntas(FilaPerguntas filaPerguntas) {
		this.filaPerguntas = filaPerguntas;
	}

	
	
	
}
